build.zig

```diff
@@ -41,28 +41,33 @@ pub fn build(b: *std.Build) !void {
             .PCRE2_MAX_VARLOOKBEHIND = 255,
             .NEWLINE_DEFAULT = 2,
             .PCRE2_PARENS_NEST_LIMIT = 250,
+
+            .PCRE2GREP_BUFSIZE = 20480,
+            .PCRE2GREP_MAX_BUFSIZE = 1048576,
```

Comment:
I'm curious why these would be needed, given that the Zig build doesn't do pcre2grep currently.

Reply:
Actually, the error messages I got I sort of 'tackled' progressively. It's possible that something I later had to implement actually over-rode the necessity. I'll test that right now.

Reply:
on the 10.45 branch (that I now realize I have always been testing on) it does seem like these are necessary to define. To what end, I can't tell, but it may be for uniformity with the rest of the `#defines` that are in `config-cmake.h.in`.

in fact, this might be the very part of the build.zig file that it turns out is unnecessary on master. I will test that.

Reply:
for the record they are also necessary on master.

Reply:
Thank you, I see that they are needed due to Zig becoming stricter. Previously it didn't complain if some substitutions were not defined - it would just substitute null/empty-string.